src/main/java/no/fintlabs/resourceserver/security/SecurityConfiguration.java

```diff
@@ -0,0 +1,185 @@
+package no.fintlabs.resourceserver.security;
+
+import no.fintlabs.resourceserver.UrlPaths;
+import no.fintlabs.resourceserver.security.client.ClientJwtConverter;
+import no.fintlabs.resourceserver.security.properties.ExternalApiSecurityProperties;
+import no.fintlabs.resourceserver.security.properties.InternalApiSecurityProperties;
+import no.vigoiks.resourceserver.security.FintJwtUserConverter;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.core.annotation.Order;
+import org.springframework.security.config.annotation.web.reactive.EnableWebFluxSecurity;
+import org.springframework.security.config.web.server.SecurityWebFiltersOrder;
+import org.springframework.security.config.web.server.ServerHttpSecurity;
+import org.springframework.security.web.server.SecurityWebFilterChain;
+import org.springframework.security.web.server.util.matcher.PathPatternParserServerWebExchangeMatcher;
+
+import java.util.List;
+
+import static no.fintlabs.resourceserver.security.client.ClientAuthorizationUtil.SOURCE_APPLICATION_ID_PREFIX;
+
+@EnableWebFluxSecurity
+//@EnableGlobalMethodSecurity(prePostEnabled = true)
```

Comment:
Skal vi ha det med eller kan det fjernes?

Reply:
Admin-API skal inn så fort som mulig, men drøyer det til alt annet er oppe og går :)

Reply:
Dette burde være med, men har ikke hatt tid til å lage tester for det, så drøyer på samme måte som med Admin-API:
Se https://github.com/FINTLabs/fint-flyt-resource-server/pull/1#discussion_r954936118